### DISCUSSION TITLE
Help please.  Stopping a failed plot

### DISCUSSION THREAD

**Discussion by neilmann** [Chia-Network/chia-blockchain#3365] [Support]
How can I stop a failed plot?  I miscalculated the space available to complete the plot and it stopped with an exception.  Now I need to recover the disk space it consumed.

**Comment by karl-johnson** [comment]
The files taking up space prior to the plot finishing are (to my knowledge) entirely in the plotting temp folder. Simply delete the all temp files from the temp location you use for plotting (assuming you have no other plots generating temp files at the moment).

**Comment by SinCatGit** [comment]
the best way for creating plots is to one tmp dir per queue.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
